rdrf/rdrf/models/definition/models.py

```diff
@@ -1463,6 +1463,7 @@ class EmailNotification(models.Model):
         (EventType.CLINICIAN_ACTIVATION, "Clinician Activation"),
         (EventType.CLINICIAN_SELECTED, "Clinician Selected"),
         (EventType.PARTICIPANT_CLINICIAN_NOTIFICATION, "Participant Clinician Notification"),
+        (EventType.PATIENT_CONSENT_CHANGE, "Patient Consent Values Change")
```

Comment:
Not a biggie but would prefer `"Patient Consent Change"` instead, so it doesn't confuse the end-users.

Reply:
Updated.